include/mbedtls/x509_crt.h

```diff
@@ -79,6 +79,8 @@ typedef struct mbedtls_x509_crt
     mbedtls_x509_sequence subject_alt_names;    /**< Optional list of Subject Alternative Names (Only dNSName supported). */
     mbedtls_x509_name hardware_module_name;     /**< Optional OtherName subject Alternative Name. */
 
+    mbedtls_x509_sequence certificate_policies; /**< Optional list of certificate4 policies (Only anyPolicy supported). */
```

Comment:
Yes, this breaks the ABI. As well as the `hardware_module_name` introduced in line 80.

Do you see another way to support these new extensions? 
I believe that in this development cycle, it is ok to break the ABI in development branch. @Patater  please confirm

Reply:
I don't know, how else this could be implemented, especially in a natural way. I only wanted to point this out, as it will matter in case of a release. As far as I understand, merging an ABI break into development, effectively makes the next release a breaking one. If this is what we plan (this is what I meant by the development cycle), then it is fine. But if we plan to have non-breaking, minor releases first, then I think this PR should be somehow tagged and the merge should be held until we are ready for an ABI breaking release.

Reply:
Thanks,yes, of course.

Reply:
The only way to avoid ABI breakage is to add yet another configuration option ( even two for the two additions?), but I don't like adding more configuration options